theme/src/theme.ts

```diff
@@ -1,9 +1,23 @@
-
 // Stylesheets
 import "./theme.scss";
-import "./resources/images/favicon.png"
+import "./resources/images/favicon.png";
 
 // Component Javascript
-import "./components/experiencefragment/change_flag.js";
 import "./components/form-options/form_dropdown.js";
 import "./components/progressbar/progressbar.js";
+import languageNavigation from "./components/languagenavigation/languagenavigation";
+import xfHeader from "./components/experiencefragment/experiencefragment_header";
+
+const documentReadyFunctions = () => {
+  languageNavigation();
+  xfHeader();
+};
+
+if (
+  document.readyState === "complete" ||
+  document.readyState === "interactive"
+) {
+  setTimeout(documentReadyFunctions, 1);
```

Comment:
Is the timeout required ? Or can you replace it with requestAnimationFrame ?

Reply:
I've updated the code to use `requestAnimationFrame()` instead of `setTimeout()`